services/configuration/src/transactions.rs

```diff
@@ -321,3 +369,38 @@ impl Transaction for Vote {
         Ok(())
     }
 }
+
+impl TxVoteUtil for VoteAgainst {
+    fn sender(&self) -> &PublicKey {
+        self.from()
+    }
+
+    fn target_cfg_hash(&self) -> &Hash {
+        self.cfg_hash()
+    }
+
+    fn into_maybe(self) -> MaybeVote {
+        MaybeVote::some(VotingDecision::VoteAgainst(self))
+    }
+}
+
+impl Transaction for VoteAgainst {
+    fn verify(&self) -> bool {
+        self.verify_signature(self.from())
+    }
+
+    fn execute(&self, fork: &mut Fork) -> ExecutionResult {
+        let _parsed_config = self.precheck(fork.as_ref()).map_err(|err| {
+            error!("Discarding dissenting vote {:?}: {}", self, err);
+            err
+        })?;
+
+        self.save(fork);
+        trace!(
+            "Put VoteAgainst:{:?} to corresponding cfg dissenting_votes_by_config_hash table",
```

Comment:
Nitpick: the text is no longer accurate.

Reply:
Done